recipe/build-cgo-platform.bat

```diff
@@ -0,0 +1,19 @@
+rem Install [de]activate scripts.
+for %%F in (activate deactivate) do (
+  if not exist "%PREFIX%\etc\conda\%%F.d" mkdir "%PREFIX%\etc\conda\%%F.d"
+  if errorlevel 1 exit 1
+
+  rem First, copy them to the work directory
+  copy "%RECIPE_DIR%\%%F-go-platform.bat" .
+  if errorlevel 1 exit 1
+
+  rem Second, expand the GOOS and GOARCH variables
+  sed -i.bak ^
+    -e "s|@GOOS@|windows|g" ^
```

Comment:
the GOOS is hard-coded to `windows` so the recipe does not depend on the `conda_goos` variant.